src/vt/context/runnable_context/lb_data.cc

```diff
@@ -78,4 +78,18 @@ typename LBData::ElementIDStruct const& LBData::getCurrentElementID() const {
   return cur_elm_id_;
 }
 
+#if vt_check_enabled(papi)
+std::unordered_map<std::string, uint64_t> LBData::getPAPIMetrics() {
+  std::unordered_map<std::string, uint64_t> papi_metrics = {};
+  for (size_t i = 0; i < papiData_->native_events.size(); i++) {
+    papi_metrics[papiData_->native_events[i]] = papiData_->values[i];
+  }
+  papi_metrics[std::string("real_time")] = papiData_->end_real_usec - papiData_->start_real_usec;
```

Comment:
I don't think we need to be explicit with `std::string` here:
```suggestion
  papi_metrics["real_time"] = papiData_->end_real_usec - papiData_->start_real_usec;
```